encoding/da.go

```diff
@@ -269,3 +269,54 @@ func TxsToTxsData(txs types.Transactions) []*types.TransactionData {
 	}
 	return txsData
 }
+
+// Fast testing if the compressed data is compatible with our circuit
+// (require specified frame header and each block is compressed)
+func TestCompressedDataCompatibility(data []byte) error {
```

Comment:
prefer renaming it to `CheckCompressedDataCompatibility`

because in golang, functions named `TestXXXX` usually indicate test code.

Reply:
Yes i have forgot about that ...